docs/how-to-contribute.md

```diff
@@ -1,3 +1,109 @@
-# How to contribute
+# Contributing to OpenEuropa
 
-_Work in progress._
+The following is a set of guidelines for contributing to OpenEuropa and its components, which are hosted in the [OpenEuropa project](https://github.com/openeuropa) GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[Before getting started](#before-getting-started)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Feature requests](#feature-requests)
+  * [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [Code styling](#code-styling)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [OpenEuropa Code of Conduct](code-of-conduct.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [TODO:add mail](mailto:TODO).
+
+## Before getting started
+
+OpenEuropa is an open source project made up of several different components. Before you report anything please make sure you are using the correct repository for the component you are referring to. 
+
+The following is a comprehensive list of components built and maintained by the OpenEuropa team: [OpenEuropa component list](openeuropa-components.md)
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report for any OpenEuropa component. Following these guidelines helps everyone involved understand your report and react accordingly.
+
+Before creating bug reports, please check [this list](#before-submitting-a-bug-report) as you might find out that you don't need to create one. When you are creating a bug report, please [include as many details as possible](#how-do-i-submit-a-good-bug-report) and use [the required template](ISSUE_TEMPLATE.md), this will allows us to resolve issues faster.
+
+
+#### Before Submitting A Bug Report
+
+* **Determine [which repository the problem should be reported in](#before-getting-started)**.
+* **Make sure the issue is really a bug and not an intended behavior** by carefully reading the component's user manual.
```

Comment:
Instead of user manual, mention component's documentation which is more generic.